src/libraries/System.Text.RegularExpressions/tests/RegexReductionTests.cs

```diff
@@ -476,7 +486,6 @@ public void PatternsReduceIdentically(string pattern1, string pattern2)
 
         [Theory]
         // Not coalescing loops
-        [InlineData("aa", "a{2}")]
```

Comment:
(Unrelated to this change) In these test methods, is there a reason why we chose not to use `Assert.Equal/NotEqual(result1, result2)` instead of manual `throw`?

Reply:
I'm sure there was at the time.  At present I can't explain it :)

Reply:
Oh, I remember why.  The built-in Assert.{Not}Equals trims the inputs to just show the first location that didn't match, plus a few characters around that.  That's actively harmful for debugging failures here.